0100/protocol.html

```diff
@@ -16,208 +16,215 @@
 -->
 
 <!--#include virtual="../includes/header.html" -->
+<!--#include virtual="../includes/top.html" -->
+<div class="content">
+	<!--#include virtual="../includes/nav.html" -->
+	<div class="right">
```

Comment:
Both of these get closed in /includes/footer.html. The old /includes/header.html used to leave these tags open and they'd get closed in footer. 

I broke header.html up a little bit so we could have more flexibility with placing page specific stuff on top of .content.